### ISSUE TITLE
MRA: двухфакторная авторизация и значок непрочитанных писем

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#2054]
>> Подскажите, пожалуйста, по плагину MRA. Миранда и плагин последние релизные.
Включил на почте mail.ru двухфакторную авторизацию. После чего (но не факт, что по этой причине) в Миранде стало некорректно работать уведомление о новых письмах.
Плагин коннектится, работает.
Шлю себе письмо — уведомления нет. Увожу MRA в оффлайн, потом в онлайн (или перезагружаю Миранду) — вот оно, уведомление, пришло. Шлю ещё письмо — опять нет и т.д. 
>>
>> Плагин понимает, что письмо пришло, он выдаёт [1] в статусе и этот символ пропадает, когда письмо становится прочитанным. См. скрин. Но плагин не показывает иконку. Иконка показывается только 1 раз — после выхода из оффлайна. Если же плагин уже в онлайне и пришло письмо — иконки нет (я проверял и на последней стабильной Миранде и в версии для разработчиков). ghazan лог выслал.

https://forum.ru-board.com/topic.cgi?forum=5&topic=50156&start=420#11

**Comment by C1eriC** [comment]
С тех пор, как я оставлял этот багрепорт прошло более года и протокол они поменяли. Но Миранда всё так же не информирует о пришедших письмах, если подключена 2FA. Проверил это только что.

**Comment by apollo2k4** [comment]
@C1eriC более того, протокол изменился и уведомления о новой почте вообще перестали приходить в независимости от настроек авторизации.

**Comment by dartraiden** [comment]
Насколько я помню, не приходят уведомления о письмах, поступивших, пока Миранда была в оффлайне.

А с 2FA ничего не изменилось.

**Comment by C1eriC** [comment]
> Насколько я помню, не приходят уведомления о письмах, поступивших, пока Миранда была в оффлайне.
Да, [всё так](http://forum.ru-board.com/topic.cgi?forum=5&topic=50156&start=1040#15) ([и тут](http://forum.ru-board.com/topic.cgi?forum=5&topic=50156&start=1060#11)). После августовского изменения протокола стало так. Раньше всегда приходили уведомления.

Думал, может если протокол поменяли, то может и с подключенной 2FA будут приходить уведомления (хотя бы пока MRA online). Но нет, так и не приходят.
